helloworld/src/components/Slider.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from "react";
+import Slide1 from "../assets/images/slider1.png"
+import Slide2 from "../assets/images/slider2.png"
+import Slide3 from "../assets/images/slider3.png"
+
+export default class Slider extends Component {
+  render() {
+    return (
+      <div id="carouselExampleControls" className="carousel slide" data-ride="carousel">
+      <div className="carousel-inner">
+        <div className="carousel-item active">
+          <img className="d-block w-100" src={Slide1} alt="First slide" />
```

Comment:
use meaningful naming for class name, what is `d-block w-100`, have you copy that from other source?

Reply:
Have you fixed it?